src/scenic/core/regions.py

```diff
@@ -1729,6 +1729,11 @@ def inradius(self):
         else:
             return region_distance
 
+    @cached_property
+    @distributionFunction
+    def circumradius(self):
+        hypot(*(self.mesh.extents / 2))
```

Comment:
Hmm, does this get used anywhere? There's no `return`, so this always returns `None`. Either this should be removed or tests for it should be added to the mesh region tests.

Reply:
Good point, removed it.